web/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+import {Fragment,useState } from 'react'
+import { Combobox } from '@headlessui/react'
+import { CheckIcon, ChevronUpDownIcon } from '@heroicons/react/20/solid'
+interface Person {
+    id: number;
+    name: string;
+  }
+interface DropdownProps {
+    people: string[]; 
+  }  
+
+function Dropdown({people}: DropdownProps) {
+  const [selectedPerson, setSelectedPerson] = useState(people[0] || '')
```

Comment:
pass these as props so that the selected option can be accessed from the parent component.